src/components/DatePicker/index.vue

```diff
@@ -212,15 +213,16 @@
                 class="square"
                 v-for="(day, dayIndexMobile) in months[n].days"
                 :key="`${datepickerDayKey}-${dayIndexMobile}`"
-                @click="handleHoveringDate(day)"
+                @mouseenter="handleHoveringDate(day)"
```

Comment:
@sbourdon13 
Because u can't use @click, because @click is on <day /> component, @mouseenter allow to avoid the bubbling

see this article to see more informations: https://javascript.info/bubbling-and-capturing